### DISCUSSION TITLE
Welcome to pvc-autoresizer Discussions!

### DISCUSSION THREAD

**Discussion by toshipp** [topolvm/pvc-autoresizer#144] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.

**Comment by Ravi-Tripathi21** [comment]
Hi,

I am currently working for Storage and K8s domain for Quality assurance and automation. I want to contribute for QA, Automation [python/Go] and also for documentation. Please let me know, Thanks, Ravi Tripathi

**Comment by umeshbitgo** [comment]
Hi,
I want to ignore a few PVCs in my cluster from being resized by pvc-autoresizer. is there an annotation to to ignore PVCs? I initially thought that if we don't annotate the PVC then it would be ignored but as per the documentation:
**The PVC can optionally have resize.topolvm.io/threshold, resize.topolvm.io/inodes-threshold and resize.topolvm.io/increase annotations. (If they are not given, the default value is 10%.)** It seems even if I don't annotate the PVC, this tool would set a default value of 10% for each of these annotations.

**Reply by ushitora-anqou** [reply]
@umeshbitgo Hi there! This is a welcome page, and we don't expect anyone to submit new questions here. Could you start a new Q&A discussion from [this page](https://github.com/topolvm/pvc-autoresizer/discussions/new/choose) instead? Thanks!
